src/Microsoft.OpenApi.Readers/V3/OpenApiTagDeserializer.cs

```diff
@@ -12,27 +13,45 @@ namespace Microsoft.OpenApi.Readers.V3
     /// </summary>
     internal static partial class OpenApiV3Deserializer
     {
+        private static readonly FixedFieldMap<OpenApiTag> _tagFixedFields = new FixedFieldMap<OpenApiTag>
+        {
+            {
+                OpenApiConstants.Name, (o, n) =>
+                {
+                    o.Name = n.GetScalarValue();
+                }
+            },
+            {
+                OpenApiConstants.Description, (o, n) =>
+                {
+                    o.Description = n.GetScalarValue();
+                }
+            },
+            {
+                OpenApiConstants.ExternalDocs, (o, n) =>
+                {
+                    o.ExternalDocs = LoadExternalDocs(n);
+                }
+            }
+        };
+
+        private static readonly PatternFieldMap<OpenApiTag> _tagPatternFields = new PatternFieldMap<OpenApiTag>
+        {
+            {s => s.StartsWith("x-"), (o, p, n) => o.AddExtension(p, n.CreateAny())}
+        };
+
         public static OpenApiTag LoadTag(ParseNode n)
         {
             var mapNode = n.CheckMapNode("tag");
 
-            var obj = new OpenApiTag();
+            var domainObject = new OpenApiTag();
```

Comment:
>domainObject [](start = 16, length = 12)

Why is this called domainObject?

Reply:
So that when I cut and paste the code from one load to the next, I don't have to rename it :-)